v3/src/components/graph/components/graph.tsx

```diff
@@ -160,7 +160,7 @@ export const Graph = observer(function Graph({graphController, graphRef, dotsRef
 
   return (
     <DataConfigurationContext.Provider value={graphModel.dataConfiguration}>
-      <div className={kGraphClass} ref={graphRef} data-testid="graph">
+      <div className={kGraphClass} ref={graphRef} data-testid="graph-plot">
```

Comment:
In the terminology of the code, the graph is the entire tile/component including axes, legend, etc. while the plot is just the area between the axes where points are plotted and adornments are shown, so this use of `plot` is unconventional.  Perhaps `graph-view` or another name if the simpler `graph` is insufficient for some reason?

Reply:
I've changed this and the test that uses it back to `graph-plot`